frontend/src/components/Layout.tsx

```diff
@@ -25,7 +25,7 @@ export default function Layout({ children, locale }: Props) {
   return (
     // Stick the footer to the bottom of the page
     <div className="display-flex flex-column minh-viewport">
-      <a className="usa-skipnav" href="#main-content">
+      <a className="usa-skipnav z-top" href="#main-content">
```

Comment:
Since the Header is now z-index'd, the skip link needs to be higher. Nothing should ever cover it.